locales/sv/routes.json

```diff
@@ -56,13 +56,17 @@
     "title": "Statistik"
   },
   "about-us": {
-    "path": "/om-oss",
```

Comment:
En tidigare fix var ju att lägga in "/" för att slugen skulle bli rätt?

Reply:
Inga andra urler i den filen har en slash, denna var felskriven.